docs/topics/control-flow.md

```diff
@@ -92,7 +92,7 @@ val numericValue = when (getRandomBit()) {
 
 In `when` _statements_, the `else` branch is mandatory in the following conditions:
 * `when` has a subject of a `Boolean`, [`enum`](enum-classes.md),
```

Comment:
Some more details for the example in the "Conversation" tab. For a non-nullable enum, an else branch is not needed (and the current wording with "or" sounds to me like it is needed). For a nullable enum, it's obligatory. This is why I think that `and` works better here.